site/sitecontent/terms.md

```diff
@@ -0,0 +1,264 @@
+
+The following terms and conditions govern all use of the Service and all content, services and products available at or through the Service. The Service is owned and operated by FOSSi Foundation. The Service is offered subject to your acceptance without modification of all of the terms and conditions contained herein and all other operating rules, policies and procedures that may be published from time to time on this website by FOSSi Foundation (collectively, the “Agreement”).
+
+Please read this Agreement carefully before accessing or using the Service. By accessing or using any part of the Service, you agree to become bound by the terms and conditions of this agreement. If you do not agree to all the terms and conditions of this agreement, then you may not access the Service or use any services. If these terms and conditions are considered an offer by FOSSi Foundation, acceptance is expressly limited to these terms.
+
+## Account Terms
+
+When using the Service to host projects, you are responsible for maintaining the security of your account and any projects, and you are fully responsible for all activities that occur under the account and any other actions taken in connection with projects.
+
+Your account must not be named in a manner that misleads other users of the Service into thinking that you are another person, organisation or company.
+
+You must not describe or assign keywords to projects in a misleading or unlawful manner, including in a manner intended to trade on the name or reputation of others, and FOSSi Foundation may change or remove any description or keyword that it considers inappropriate or unlawful, or otherwise likely to cause FOSSi Foundation liability. You must immediately notify FOSSi Foundation of any unauthorized uses of your projects, your account or any other breaches of security. FOSSi Foundation will not be liable for any acts or omissions by You, including any damages of any kind incurred as a result of such acts or omissions.
+
+## Responsibility of Contributors
+
+If you maintain a project, contribute to a project, post material to the Service, post links on the website, or otherwise make (or allow any third party to make) material available by means of the Service (any such material, “Content”), You are entirely responsible for the content of, and any harm resulting from, that Content. That is the case regardless of whether the Content in question constitutes text, graphics, a computer-aided design or computer-aided manufacturing file, an audio file, computer software or a programmable logic bitstream. By making Content available, you represent and warrant that:
+
+* the downloading, copying and use of the Content will not infringe the proprietary rights, including but not limited to the copyright, patent, trademark or trade secret rights, of any third party;
```

Comment:
This sounds rather broad and possibly unachievable to me. Does this mean: only for content I personally created I need to warrant that it doesn't infringe proprietary rights, or also for content others created? Otherwise, this clause would mean I personally become responsible for code is infringing, for example, patents and which I added to LibreCores.